### ISSUE TITLE
mWeb(android) - Chrome: Inconsistency in `Lint warning sound` (way too low) 

### ISSUE THREAD

**Issue by PiyushChandra17** [processing/p5.js-web-editor#2725]
### p5.js version

2.10.0

### What is your operating system?

None

### Actual Behavior

mWeb(android) - Chrome: Inconsistency in `Lint warning sound` (way too low) 

### Expected Behavior

mWeb(android) - Chrome: `Lint warning sound` should be consistent across all web and mobile browsers

### Steps:
1. Open the editor web app in android - mweb `chrome`
2. Click on three dots > accessibility
3. Tap on preview sound
4. Notice the sound is way too low as compared with web browsers

**Comment by lindapaiste** [comment]
I have no idea how we would control the volume of an audio track, but feel free to investigate this.

**Comment by KrishavRajSingh** [comment]
How much volume will be perfect? like 60% or 70% or 80% ??

**Comment by PiyushChandra17** [comment]
@KrishavRajSingh I think should be consistent throughout

**Comment by KrishavRajSingh** [comment]
Yes, but different devices have different strength of speaker( like laptop has speaker with more strength as compared to mobile phones)
